services/net/cpnews/appsettings.json

```diff
@@ -0,0 +1,44 @@
+{
+  "BaseUrl": "/",
+  "Logging": {
+    "LogLevel": {
+      "Default": "Warning",
+      "Microsoft.AspNetCore": "Warning"
+    }
+  },
+  "AllowedHosts": "*",
+  "Service": {
+    "MaxFailLimit": 5,
+    "ApiUrl": "http://api:8080",
+    "MediaTypes": ["CPNews"],
+    "TimeZone": "Pacific Standard Time"
+  },
+  "Auth": {
+    "Keycloak": {
```

Comment:
Checkout the latest changes in my prior PR for configuration. There are some differences in the Keycloak section.